main.py

```
@@ -163,15 +151,31 @@ def main() -> None:
+                    f"Tenant '{tenant_name}' could not be created. Cannot create entities for map '{map_file_path}'"
+                )
+                return
+            api_client.set_default_header("Cookie", tenant_cookie)
+
+            if args.delete and tenant_name not in already_deleted_tenants:
+                delete_all(api_client)
+                already_deleted_tenants.append(tenant_name)
+                print(f"Entries for tenant {tenant_name} deleted")
+
+            run_queries(api_client, map_config, already_added_cars)
         except Exception as exception:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)